test/lev-report/charts/HourlyUsage.test.js

```diff
@@ -35,4 +36,21 @@ describe('HourlyUsage component', () => {
 		expect(tree).toMatchSnapshot();
 		success = true;
 	});
+
+	afterAll(() => {
+		if (success && process.argv.includes('--visualise')) {
+			const html = `<html>
+			<head>
+					<title>Component Preview</title>
+					<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+			</head>
+			<body style="margin:0;">
+					<div id="app">
+						 ${renderToString(<HourlyUsage traces={data}/>)}
+					</div>
+			</body>
+			</html>`;
+			require('fs').writeFileSync('test/lev-report/charts/__snapshots__/HourlyUsage.html', html);
```

Comment:
Does this need an entry in `.gitignore`?

Reply:
Yes, good shout!

Reply:
Does it though. I thought I spotted you testing against a previous snapshot later on in the code. If that is the case it would need to be committed. (Although I'm not certain of the value of that sort of testing.)

Reply:
This file should be ignored (or at least can be), because it's not used for the testing, it just allows   components to be viewed.

Reply:
Okay.